CDP4Common/Poco/Thing.cs

```diff
@@ -1,789 +1,790 @@
-        /// </summary>
-        [UmlInformation(aggregation: AggregationKind.None, isDerived: false, isOrdered: false, isNullable: false, isPersistent: true)]
-        [DataMember]
-        public int RevisionNumber { get; internal set; }
-
-        /// <summary>
-        /// The data-source's <see cref="Uri"/> this <see cref="Thing"/> comes from
-        /// </summary>
-        [Browsable(true)]
-        public Uri IDalUri { get; internal set; }
-
-        /// <summary>
-        /// Gets the Cache that contains the current <see cref="Thing"/>
-        /// </summary>
-        public ConcurrentDictionary<CacheKey, Lazy<CommonData.Thing>> Cache { get; set; }
-
-        /// <summary>
-        /// Gets or sets the Container of the current Thing
-        /// </summary>
-        public Thing Container { get; set; }
-
-        /// <summary>
-        /// Gets the original <see cref="Thing"/> form which the current <see cref="Thing"/> has been cloned.
-        /// </summary>
-        /// <remarks>
-        /// If the <see cref="Thing"/> is not a clone, this property returns null
-        /// </remarks>
-        public Thing Original { get; protected set; }
-
-        /// <summary>
-        /// Gets the user-friendly name for this <see cref="Thing"/>
-        /// </summary>
-        public virtual string UserFriendlyName
-        {
-            get { return "User-friendly name not implemented."; }
-        }
-
-        /// <summary>
-        /// Gets the user-friendly short-name for this <see cref="Thing"/>
-        /// </summary>
-        public virtual string UserFriendlyShortName
-        {
-            get { return "User-friendly short-name not implemented."; }
-        }
-
-        /// <summary>
-        /// Gets the key with which the current <see cref="Thing"/> is stored in the cache
-        /// </summary>
-        public CacheKey CacheKey
-        {
-            get
-            {
-                if (this.cacheKey.Thing != Guid.Empty)
-                {
-                    return this.cacheKey;
+        /// <summary>
+        /// Gets or sets a list of <see cref="Person"/> that are excluded for this instance.
+        /// </summary>
+        /// <remarks>
+        /// Excluded <see cref="Person"/> do not have access to the instance
+        /// </remarks>
+        [CDPVersion("1.1.0")]
+        [UmlInformation(AggregationKind.None, false, false, false, true)]
+        [DataMember]
+        public virtual List<Person> ExcludedPerson { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Universally Unique Identifier (UUID) that uniquely identifies an instance of <see cref="Thing"/>
+        /// </summary>
+        [Browsable(true)]
+        [UmlInformation(AggregationKind.None, false, false, false, true)]

```

Comment:
can you please restore the explicit named parameters when initializing the `UmlInformation` annotation